src/box/lua/tuple.lua

```diff
@@ -112,6 +112,7 @@ local encode_array = msgpackffi.internal.encode_array
 local encode_r = msgpackffi.internal.encode_r
 
 local tuple_encode = function(tmpbuf, obj, level)
+    local tuple_offset = tmpbuf.wpos - tmpbuf.rpos
```

Comment:
Nit. Let's change to `used = tmpbuf.size()`. It is more consistent with the other code and has better usage of `ibuf` API.